sass/elements/box.sass

```diff
@@ -15,7 +16,11 @@ $box-link-active-shadow: inset 0 1px 2px rgba($black, 0.2), 0 0 0 1px $link !def
   color: $box-color
   display: block
   padding: $box-padding
-
+  &.is-small
+    padding: $box-padding-small
+  &.is-rounded
+    border-radius: $radius-rounded
+    
```

Comment:
You should remove the trailing whitespaces here so the project is more consistent following the *Bulma Sass styleguide*.